ci/build.yml

```diff
@@ -274,6 +275,7 @@ connect-web build:
       - packages/connect-examples/webextension-mv2/build-legacy
       - packages/connect-examples/webextension-mv3/build
       - packages/connect-examples/webextension-mv3/build-legacy
+      - packages/connect-explorer-webextension/build
```

Comment:
nice, not now, but in the future  Iwould like to remove builds based on examples and extend connect-explorer-webextension builds with all those legacy builds instead.